### PR TITLE
KATI-83_Book-Appointment-Create-API

### PR DESCRIPTION
**Pull request by vtnlinh95**

### CODE REVIEW

File: com.kms.cura.dal/src/com/kms/cura/dal/database/AppointmentDatabaseHelper.java [hunk 1]
```diff
@@ -112,10 +116,51 @@ private PreparedStatement getAppointmentQuery(AppointmentEntity entity) throws S
 		if (entity.getApptDay() != null) {
 			columnValueMap.put(AppointmentColumn.STATUS.getColumnName(), entity.getStatus());
 		}
+		if(entity.getStartTime() != null){
```
Comment (hieumai) [inline com.kms.cura.dal/src/com/kms/cura/dal/database/AppointmentDatabaseHelper.java]: re-format this file

**Comment by hieumai** [comment]
Looks OK to me. @huuduy216  and @toannguyen71195  please start verifying on this

**Comment by vtnlinh95** [comment]
@toannguyen71195  @huuduy216  the json file for testing 
[json to test create.docx](https://github.com/hieumai/cura/files/363245/json.to.test.create.docx)

url : http://localhost:8080/com.kms.cura_server/cura_server/appt/createAppts

**Comment by hieumai** [comment]
@vtnlinh95  rebase this so I can merge

**Pull request merged**